tests/unit_tests/fault/test_fault.py

```diff
@@ -34,13 +34,13 @@ def test_add_connection_set_and_tmults(example_model_with_properties, test_data_
     reload_model = rq.Model(epc_file = model.epc_file)
 
     faults = reload_model.parts_list_of_type('obj_FaultInterpretation')
-    assert len(faults) == len(expected_mult.keys()),  \
+    assert len(faults) == len(expected_mult.keys()), \
```

Comment:
It would be nice if you could set up your IDE not to make these sort of changes to existing code. Otherwise I will have to figure out how to configure mine to change the code back next time I am working on the module!